src/components/betting-chips/BettingChip.tsx

```diff
@@ -1,53 +1,24 @@
 import React from "react";
-import { observer } from "mobx-react";
-import sound from "./betting-chip-sound.mp3";
 
 import styles from "./BettingChip.module.scss";
-import { useMainStore } from "../../hooks/useMainStore";
-import { useAudio } from "../../hooks/useAudio";
 
 export type BettingChipsValues = 1 | 5 | 25 | 50 | 100;
 export type BetOnOptions = "player" | "tie" | "bank";
 
 export type BettingChipProps = {
   value: BettingChipsValues;
-  betOn: BetOnOptions;
+  isHidden: boolean;
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
You can change it to pass value instead of event. Then in `BettingChips` component code will be simplified